admin/src/components/LinksTransactionListFormular.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="links-transaction-list-formular">
+    {{ $t('transactionlink.form_header') }}
+    <b-table striped hover :fields="fields" :items="items"></b-table>
```

Comment:
No table when `items.length === 0` instead a message that there are no links

Reply:
no table if items.length  === 0 ( v-if items.length > 0 )